lint/ruff.bzl

```diff
@@ -1,13 +1,24 @@
-"""API for declaring a Ruff lint aspect that visits py_library rules.
+"""API for declaring a Ruff lint aspect that visits py_{binary|library|test} rules.
 
 Typical usage:
 
+Ruff is provided as a built-in tool by rules_lint. To use the built-in version, first add a dependency on rules_multitool to MODULE.bazel:
```

Comment:
drive-by question since you're current on this code, I wonder if we can remove the https://github.com/aspect-build/rules_lint/blob/main/lint/ruff_versions.bzl file

Reply:
Looks like it's only used by the workspace version of the rules, [here](https://github.com/aspect-build/rules_lint/blob/2dce2664821535ff581ebe6c93022f8273cda997/lint/ruff.bzl#L273). Depends whether you want to keep workspace support.